src/components/Tabs/Tabs.jsx

```diff
@@ -1 +1,41 @@
-export const Tabs = () => {};
+export const Tabs = ({ tabs, selectedTabId, onTabSelected }) => {
+  let checkedTabId = selectedTabId;
+
+  if (tabs.find(searchedTab => searchedTab.id === checkedTabId) === undefined) {
+    checkedTabId = tabs[0].id;
+  }
+
+  const activeTab = tabs.find(searchedTab => searchedTab.id === checkedTabId);
```

Comment:
You can simplify this just by writing
```js
const selectedTab = tabs.find(tab => selectedTabId === tab.id) || tabs[0];
```
After that, we'll have only 1 variable `selectedTab`

Just my preferences, not critical

Reply:
@Drach-Illya FYI please, let's just use `.find()` method